### PR TITLE
thumbnail path in media file object should now be set whether the thu…

### PR DESCRIPTION
**Pull request by mnbogner**
…mbnail file is loaded or created

### CODE REVIEW

File: lib/src/main/java/scal/io/liger/MediaHelper.java [hunk 1]
```diff
@@ -67,10 +67,10 @@
      */
     public static abstract class ThumbnailCallback {
         /** Called when a thumbnail request requires generation of a new thumbnail */
-        public void newThumbnailGenerated(File thumbnail) {}
+        public abstract void newThumbnailGenerated(File thumbnail);
```
Comment (vitriolix) [inline lib/src/main/java/scal/io/liger/MediaHelper.java:70]: Why did you need to make this change?

Comment (mnbogner) [inline lib/src/main/java/scal/io/liger/MediaHelper.java:70]: i wanted to force classes that extended the callback to implement the methods for both generating and loading thumbnails.  the problem was invisible because even though the actual callback had no method to handle generated thumbnails, this empty method was called when a thumbnail was generated.
